digraph "CFG forkotlin.coroutines.experimental.SequenceBuilderIterator.resumeWithException(Ljava\/lang\/Throwable;)V" {
Node_0 [shape=record,label="{0\:\ 0x0000|MTH_ENTER_BLOCK\l|0x0000: CONST_STR  (r0 I:java.lang.String) =  \"exception\" \l0x0002: INVOKE  (r2 I:java.lang.Object), (r0 I:java.lang.String) kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull(java.lang.Object, java.lang.String):void type: STATIC \l}"];
Node_1 [shape=record,label="{1\:\ 0x0005|0x0005: THROW  (r2 I:java.lang.Throwable) \l}"];
MethodNode[shape=record,label="{public void kotlin.coroutines.experimental.SequenceBuilderIterator.resumeWithException((r1 'this' I:kotlin.coroutines.experimental.SequenceBuilderIterator A[IMMUTABLE_TYPE, THIS]), (r2 I:java.lang.Throwable A[IMMUTABLE_TYPE, METHOD_ARGUMENT]))  | Annotation[BUILD, org.jetbrains.annotations.NotNull, \{\}]\l}"];
MethodNode -> Node_0;
Node_0 -> Node_1;
}

